app/decorators/ncr/work_order_decorator.rb

```diff
@@ -0,0 +1,28 @@
+module Ncr
+  class WorkOrderDecorator < Draper::Decorator
+    delegate_all
+
+    EMERGENCY_APPROVER_EMAIL = 'Emergency - Verbal Approval'
+    NO_APPROVER_FOUND        = 'No Approver Found'
```

Comment:
I know I mentioned this in another PR, but not sure what the outcome was. Do we want to be lining up `=` ? Seems like we are being inconsistent on that here. I would vote for no.